pkg/module_manager/global_hook_config.go

```diff
@@ -283,38 +291,33 @@ func NewHookConfigFromGoConfig(input *sdk.HookConfig) config.HookConfig {
 		monitor.WithFieldSelector(kubeCfg.FieldSelector)
 		monitor.WithNamespaceSelector(kubeCfg.NamespaceSelector)
 		monitor.WithLabelSelector(kubeCfg.LabelSelector)
-		monitor.JqFilter = kubeCfg.JqFilter
-		monitor.FilterFunc = kubeCfg.FilterFunc
+		monitor.FilterFunc = kubeCfg.Filterable.ApplyFilter
```

Comment:
`kubeCfg.Filterable` should be checked for nil.

Reply:
Done.